packages/edge-bundler/tsconfig.json

```diff
@@ -5,7 +5,7 @@
     /* Basic Options */
     "incremental": true /* Enable incremental compilation */,
     "target": "ES2019" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */,
-    "module": "es2020" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', 'es2020', or 'ESNext'. */,
+    "module": "nodenext" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', 'es2020', or 'ESNext'. */,
```

Comment:
I was getting some errors when importing `@netlify/edge-functions-bootstrap` with the previous `module` and `moduleResolution` settings. I think these changes should be safe, though?